desktop/ui/src/main/java/org/datacleaner/widgets/ExecuteButtonBuilder.java

```diff
@@ -38,11 +40,21 @@
 import org.datacleaner.util.WidgetUtils;
 import org.datacleaner.widgets.ExecuteButtonOptions.ExecutionMenuItem;
 import org.datacleaner.windows.AnalysisJobBuilderWindow;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * A builder for the "Execute" button in the job builder window of DataCleaner
  */
 public class ExecuteButtonBuilder {
+    private static final Logger LOG = LoggerFactory.getLogger(ExecuteButtonBuilder.class);
+    
+    /**
+     * Property which may contain a comma separated vector of class names
+     * implementing the ExecutionMenuItem interface, which are added as extra
+     * execute buttons to the job builder window of DataCleaner.
+     */
+    public static final String EXTRA_BUTTON_CLASSES = "datacleaner.ui.desktop.extra.execute.buttons";
```

Comment:
Shouldn't this be called something with "menuitem" instead of "button"? Or maybe just "options" to make it less widget-specific?